specifications/api/swagger.yaml

```diff
@@ -1311,6 +1316,20 @@ paths:
           schema: *parametersQuitStakePool
       responses: *responsesQuitStakePool
 
+  /stake-pools/{stakePoolId}/wallets/{walletId}/fee:
+    get:
+      operationId: joinStakePoolFee
+      tags: ["Stake Pools"]
+      summary: Estimate
+      description: |
+        <p align="right">status: <strong>stable</strong></p>
+
+        Estimate fee for joining a stake pool.
+      parameters:
+        - *parametersStakePoolId
```

Comment:
Why is this parameter required?

See https://github.com/input-output-hk/cardano-wallet/pull/1116/files#diff-610bd6b69383f967d3fad205d6c6d86aR854